portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsUploadField.testcase

```diff
@@ -462,6 +711,117 @@ definition {
 
 	}
 
+	@description = "This is a use case for LPS-125499."
+	@ignore = "true"
```

Comment:
@leticiachiu, can you please clarify why this test is ignored? Is it passing locally? If not, why?